aspnet/fundamentals/configuration.rst

```diff
@@ -15,25 +15,24 @@ ASP.NET Core supports a variety of different configuration options. Application
 Getting and setting configuration settings
 ------------------------------------------
 
-ASP.NET Core's configuration system has been re-architected from previous versions of ASP.NET, which relied on ``System.Configuration`` and XML configuration files like ``web.config``. The new :doc:`configuration model </fundamentals/configuration>` provides streamlined access to key/value based settings that can be retrieved from a variety of sources. Applications and frameworks can then access configured settings using the new :ref:`Options pattern <options-config-objects>`
+ASP.NET Core's configuration system has been re-architected from previous versions of ASP.NET, which relied on ``System.Configuration`` and XML configuration files like ``web.config``. The new configuration model provides streamlined access to key/value based settings that can be retrieved from a variety of sources. Applications and frameworks can then access configured settings using the new :ref:`Options pattern <options-config-objects>`
```

Comment:
Consider mentioning that Options is provides a strongly typed mechanism to access configuration settings as an advantage, e.g. for the last sentence, you could say something like:

> Applications and frameworks can then access configured settings in a strongly typed fashion using the new :ref:`Options pattern <options-config-objects>`